src-tauri/src/farmer.rs

```diff
@@ -206,11 +208,17 @@ async fn farm(base_directory: PathBuf, farm_args: FarmingArgs) -> Result<(), any
         match result {
             Err(error) => {
                 error!("{error}");
-                error_sender.send(()).await.unwrap() // this send should always be successful
+                error_sender
+                    .send(())
+                    .await
+                    .expect("error receiver is always listening")
```

Comment:
I actually care about the `receiver` being alive. The `receiver` is in `farming` function, and it should always be alive. That's why I put `expect` here. 
I get that it is not obvious from the context of the function. But it should be clear after inspecting `farming` function. 

I will put a more descriptive expect message. If you still think it is not a good idea, I can use error handling in here, because I want to be sure that `receiver` is alive.